http.go

```diff
@@ -117,6 +130,59 @@ func (r *httpRequest) encodeBody() (io.Reader, error) {
 	}
 }
 
+func (r httpRequest) isMultipartFormDataMediaType() bool {
+	if r.mediaType == MediaTypeMultipartFormData {
+		return true
+	}
+	return strings.HasPrefix(r.mediaType, MediaTypeMultipartFormData+"; boundary=")
+}
+
+func (r *httpRequest) encodeMultipart() (io.Reader, error) {
+	quoteEscaper := strings.NewReplacer("\\", "\\\\", `"`, "\\\"")
+	values, ok := r.body.(map[string]interface{})
+	if !ok {
+		return nil, fmt.Errorf("invalid body: %v", r.body)
+	}
+	buf := &bytes.Buffer{}
+	mw := multipart.NewWriter(buf)
+	if os.Getenv("TEST_MODE") == "true" {
```

Comment:
How about creating `httpRequest.boundary` and injecting values as needed?

Reply:
Is this what you mean? I'd like to fix boundary in `TestRunbook` so I cannot access `httpRequest`.
```
type httpRequest struct {
	path      string
	method    string
	headers   map[string]string
	mediaType string
	body      interface{}

	multipartWriter *multipart.Writer
	boundary        string
}
...
buf := &bytes.Buffer{}
mw := multipart.NewWriter(buf)
if r.boundary != "" {
  _ = mw.SetBoundary(r.boundary)
  //_ = mw.SetBoundary("123456789012345678901234567890abcdefghijklmnopqrstuvwxyz")
}
```

Or what you mean is supporting boundary via yaml book?
```
req:
  /upload:
    post:
      body:
        multipart/form-data:
          boundary: 1234567890abcdefg
          field1: '/path/to/image1.png'
          field2: '/path/to/image2.png'
```

Reply:
> in TestRunbook so I cannot access httpRequest

Oh, I see.

To inject boundary, we need to pass the value via HTTP Runner with httpRunnerOption.

I'll try to fix it at another time. Thank you 👍